### PR TITLE
Feature/card display

### PR DESCRIPTION
**Pull request by nickhartdev**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Testing - building tests
- [ ] Formatting - no new features
- [ ] Styling- no new features
### Detailed Description (Why is this change required? What problem does it solve? Any challenges that arose while implementing this feature? If so, how were the addressed?)
- Organizes card display by clothing type
- Fixes sporty image URLs to properly display the images

### CODE REVIEW

File: code/web/src/modules/survey/api/state.js [hunk 1]
```diff
@@ -21,31 +20,28 @@ const surveyInitialState = {
 export const surveyInfo = (state = surveyInitialState, action) => {
   switch (action.type) {
     case GET_CLOTHING:
+      const types = action.products.reduce((typeArray, product) => {
```
Comment (AurumValian) [inline code/web/src/modules/survey/api/state.js:23]: This is so useful!

**Review by AurumValian — APPROVED** [review]
Good job cleaning a lot up!

**Review by Chulstro — APPROVED** [review]
Looks good! Apologies that the image files were giving you hassle

**Review by ejdelsztejn — APPROVED** [review]
Thanks for fixing this!

**Pull request merged**